starter/tslint.json

```diff
@@ -1,56 +1,43 @@
 {
-  "extends": [
-    "tslint:recommended"
-  ],
-  "rulesDirectory": [
-    "node_modules/codelyzer"
-  ],
-  "rules": {
-    // Custom
-    "trailing-comma": [false, {"multiline": "always", "singleline": "never"}],
-    "interface-name": [false, "always-prefix"],
-    // Angular 2
-    "component-class-suffix": true,
-    // "component-selector": [true, "element", "my", "kebab-case"],
-    "directive-class-suffix": true,
-    // "directive-selector": [true, "attribute", "my", "camelCase"],
-    "import-destructuring-spacing": true,
-    "no-attribute-parameter-decorator": true,
-    "no-forward-ref": true,
-    "no-input-rename": true,
-    "no-output-rename": true,
-    "only-arrow-functions": false,
-    "pipe-naming": [true, "camelCase", "my"],
-    "use-host-property-decorator": true,
-    "use-input-property-decorator": true,
-    "use-life-cycle-interface": true,
-    "use-output-property-decorator": true,
-    "use-pipe-transform-interface": true,
-    // General
-    "no-console": [true,
-      "time",
-      "timeEnd",
-      "trace"
-    ],
-    "max-line-length": [
-      true,
-      100
-    ],
-    "no-string-literal": false,
-    "no-use-before-declare": true,
-    "object-literal-sort-keys": false,
-    "ordered-imports": false,
-    "quotemark": [
-      true,
-      "single",
-      "avoid-escape"
-    ],
-    "variable-name": [
-      true,
-      "allow-leading-underscore",
-      "allow-pascal-case",
-      "ban-keywords",
-      "check-format"
-    ]
-  }
+	"extends": ["tslint:recommended", "tslint-config-prettier"],
+	"rulesDirectory": ["node_modules/codelyzer"],
+	"rules": {
+		// Custom
+		"interface-name": [false, "always-prefix"],
+		// Angular 2
+		"component-class-suffix": true,
+		// "component-selector": [true, "element", "my", "kebab-case"],
+		"directive-class-suffix": true,
+		// "directive-selector": [true, "attribute", "my", "camelCase"],
+		"no-attribute-parameter-decorator": true,
+		"no-forward-ref": true,
+		"no-input-rename": true,
+		"no-output-rename": true,
+		"only-arrow-functions": false,
+		"pipe-naming": [true, "camelCase", "my"],
+		"use-host-property-decorator": true,
+		"use-input-property-decorator": true,
+		"use-life-cycle-interface": true,
+		"use-output-property-decorator": true,
+		"use-pipe-transform-interface": true,
+		// General
+		"no-console": [true, "time", "timeEnd", "trace"],
+		"no-string-literal": false,
+		"no-use-before-declare": true,
+		"object-literal-sort-keys": false,
+		"ordered-imports": false,
+		"variable-name": [true, "allow-leading-underscore", "allow-pascal-case", "ban-keywords", "check-format"]
+	}
+	//These rules are handled by Prettier
```

Comment:
If I understand correctly, we will still get the linting warnings/errors in IntelliJ/webpack for these rules right?